.rubocop.yml

```diff
@@ -0,0 +1,70 @@
+inherit_gem:
+  jekyll: .rubocop.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+
+Metrics/LineLength:
+  Exclude:
+    - lib/mercenary/command.rb
+    - lib/mercenary/presenter.rb
+    - lib/mercenary/program.rb
+    - examples/**/*
+    - spec/**/*
+    - mercenary.gemspec
+
+Metrics/BlockLength:
+  Exclude:
+    - examples/**/*
+    - spec/**/*
+
+Style/FrozenStringLiteralComment:
+  Enabled: true
+  EnforcedStyle: always
+  Exclude:
+    - lib/mercenary/command.rb
+
+Style/Documentation:
+  Enabled: false
+
+Style/MethodMissing:
+  Exclude:
+    - lib/mercenary/presenter.rb
+
+Naming/FileName:
+  Exclude: false
```

Comment:
perhaps you meant `Enabled: false` here? :smiley:

Reply:
Yes and I realize we don't need this here, it's already defined in Jekyll's config.